docs/experimental.md

```diff
@@ -128,3 +128,38 @@ Set the location of your manually downloaded database by following the instructi
 ### Limitations
 
 1. Commit level scanning is not supported. 
+
+## License scanning
+
+OSV-Scanner supports license checking as an experimental feature. The data comes from the [deps.dev API](https://docs.deps.dev/api/v3alpha/).
+
+### License summary
+
+If you want a summary of your dependencies licenses, use the `--experimental-license-summary` flag:
+
+```bash
+osv-scanner --experimental-license-summary path/to/repository`
```

Comment:
the flag is pluralised "licenses" everywhere ("experimental-licenses-summary", "experimental-licenses"). do you think this is weird? should be singular "license"? I can't tell anymore from having worked on this feature for so long.

Reply:
Weirdly I think `--experimental-licenses` makes sense and `--experimental-license-summary` makes sense but we should definitely not have one plural and the other singular. Let's stick with plural. I will fix it.